modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/index/ComplexSecondaryKeyUnwrapSelfTest.java

```diff
@@ -106,30 +103,34 @@ public void testSecondaryIndexSimplePk() {
      *
      * @param tblName name of table which should be checked to using secondary indexes.
      * @param nameVal Value for name param.
+     * @param expResCnt Expceted result count.
+     * @param assertLabel Assert label.
      */
-    private void checkUsingIndexes(String tblName, String nameVal) {
+    private void checkUsingIndexes(String tblName, String nameVal, int expResCnt, String assertLabel) {
```

Comment:
assertLabel has to be replaced with abbreviation. Please check [1]. It can be configured for IDE.

[1] https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules

Reply:
Thanks for the link, I'll fix it